clinica/utils/input_files.py

```diff
@@ -0,0 +1,128 @@
+# coding: utf8
+
+"""
+Describe files to grab, to use with inputs.clinica_file_reader() and inputs.clinica_group_reader()
+
+"""
+
+""" T1w """
+
+# BIDS
+
+T1W_NII = {'pattern': 'sub-*_ses-*_t1w.nii*',
+           'description': 'T1w MRI'}
+
+# FreeSurfer
+
+T1_FS_WM = {'pattern': 'sub-*_ses-*/mri/wm.seg.mgz',
+            'description': 'segmentation of white matter (mri/wm.seg.mgz)',
+            'needed_pipeline': 't1-freesurfer'}
+
+T1_FS_BRAIN = {'pattern': 'sub-*_ses-*/mri/brain.mgz',
+               'description': ' extracted brain from T1w MRI',
+               'needed_pipeline': 't1-freesurfer'}
+
+T1_FS_ORIG_NU = {'pattern': 'mri/orig_nu.mgz',
+                 'description': 'intensity normalized volume generated after correction for'
+                                ' non-uniformity in FreeSurfer (file .../mri/orig_nu.mgz) ',
+                 'needed_pipeline': 't1-freesurfer'}
+
+T1_FS_WS_R = {'pattern': 'sub-*_ses-*/surf/rh.white',
```

Comment:
`T1_FS_WS_R` should be coherent with `T1_FS_WM` defined above. What about something like:
- `T1_FS_R_WM_SURF` : this would be coherent with the [WIP BEP11](https://docs.google.com/document/d/1YG2g4UkEio4t_STIBOqYOwneLEs1emHIXbGKynx7V0Y/edit#) 
- `T1_FS_WM_SURF_R` 
- `T1_FS_SURF_WM_R` 
?

Reply:
See above